public/locales/en/common.json

```diff
@@ -237,7 +237,14 @@
   "pleaseCheckYourEmail": "Please check your emails (Step 2 of 3)",
   "unsubscribeConfirmation": "Please click the confirmation link in the email within 24 hours to finish unsubscribing.",
   "projectsTitle": "Explore our projects",
-  "projectsText": "Upcoming projects provide some initial information about our new ideas, so that you can see what we’re working on. Current projects are interactive, so that you can explore and test them out.\n\nYou will soon be able to provide anonymous feedback on current projects.\n\nYou can sign up to get invited to research sessions.",
+  "projectsIntro": "Our projects provide a preview of government tools and services that are works-in-progress.",
+  "projectsText": "<strong>Upcoming projects</strong> provide a preview of tools and services we are working to make available for your feedback.\n\n<strong>Current projects</strong> are interactive, and you can provide <strong>anonymous feedback</strong> as you explore and test them out.\n\n",
+  "projectQuestions": "As you explore our current projects, keep in mind some of the following questions for feedback:\n\n",
+  "projectQuestion1": "Is the information being provided easy to understand?",
+  "projectQuestion2": "Did you expect the website or app to look or interact differently?",
+  "projectQuestion3": "Do you feel confident navigating the system?",
+  "projectQuestion4": "Would you need the help of a technical person to be able to use this system?",
+  "projectSignup": "\nIf you are interested in participating in our research tests and interviews, you can <a href='/signup' style='text-decoration: underline; color: rgba(43, 67, 128, 1);'>sign up to join our participant pool</a>, and we will contact you about projects based on eligibility.",
```

Comment:
I would really prefer not to use `\n` for newlines. We should be using css to add space, and actual <p> tags.